static/scss/resource.scss

```diff
@@ -4,7 +4,7 @@ html {
   scroll-behavior: smooth;
 }
 
-.tos-banner {
+.resource-banner {
   width: 100%;
   padding: 75px 0;
   background: rgba(53, 53, 53, .97) url(/static/images/banner-bg.png) repeat-x 0 0;
```

Comment:
Can you please reference the `$static-path` variable from `variables.scss` here?